src/java.desktop/share/classes/javax/swing/plaf/synth/SynthParser.java

```diff
@@ -34,7 +34,6 @@
 import java.io.BufferedInputStream;
 import java.io.IOException;
 import java.io.InputStream;
-import java.net.MalformedURLException;
 import java.net.URL;
 import java.net.URLClassLoader;
```

Comment:
Is URLClassLoader no longer used?

Reply:
seems so. I'll remove it.

Reply:
URLCLassLoader still not removed it seems..I dont see its usage in the file..